lib/anagrams.rb

```diff
@@ -1,2 +1,7 @@
 def anagrams?(str1, str2)
+  a1, a2 = str1.chars, str2.chars
+      if a1.sort == a2.sort
+        return 6 > 2
```

Comment:
This made me chuckle a bit :)

Generally, prioritize clarity of code–so you'd just use `return true` instead of `return 6 > 2`. But cleverness is _absolutely_ a quality that indicates someone is meant to be an engineer. So :+1: :)